apps/server/src/modules/task/service/task-copy.service.spec.ts

```diff
@@ -410,8 +403,8 @@ describe('task copy service', () => {
 
 			const setupWithFiles = () => {
 				const school = schoolFactory.build();
-				const file1 = fileFactory.buildWithId({ name: 'file1.jpg' });
-				const file2 = fileFactory.buildWithId({ name: 'file2.jpg' });
+				const file1 = { id: new ObjectId().toHexString(), name: 'file1.jpg' };
+				const file2 = { id: new ObjectId().toHexString(), name: 'file2.jpg' };
```

Comment:
fileRecordFactory

Reply:
Duplicate of https://github.com/hpi-schul-cloud/schulcloud-server/pull/4303#discussion_r1305532303 , let's continue the discussion there.